### PR TITLE
Reduce default accept queue size to 10

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

**Review by gwittel — APPROVED** [review]
LGTM

**Review by nr17 — APPROVED** [review]
What's the default number of worker threads handling the connections? If we are reducing it to 10, hopefully the workers aren't going to be starved of work and thereby reducing the effective throughput.

**Comment by johngmyers** [comment]
This limits the number of new connections that can queue before having a worker thread call accept().

File: http-server/src/main/java/com/proofpoint/http/server/HttpServerConfig.java [hunk 1]
```diff
@@ -61,9 +61,9 @@
       The OS can truncate the accept queue size to the platform defaults.
       E.g. for linux, the effective server listen backlog queue size cannot be set greater
       than /proc/sys/net/core/somaxconn on the system.
-      If you feel the need to increase httpAcceptQueueSize, increase the OS limit first.
+      If you feel the need to increase httpAcceptQueueSize, you might need to increase the OS limit first.
     */
-    private int httpAcceptQueueSize = 128;
+    private int httpAcceptQueueSize = 10;
```
Comment (therontock) [inline http-server/src/main/java/com/proofpoint/http/server/HttpServerConfig.java:66]: What about warning the user if this value is > 128?  It's anyway going to be ignored, and setting it to a large value is inviting the kind of fail-to-make-progress problem that we saw with OMS-4005.

Comment (johngmyers) [reply http-server/src/main/java/com/proofpoint/http/server/HttpServerConfig.java:66]: The comment suggests that with an OS configuration change the value could be >128.

Comment (therontock) [reply http-server/src/main/java/com/proofpoint/http/server/HttpServerConfig.java:66]: Very few people will see the comment in the code.  But a warning might get someone to notice.

**Pull request merged**